crates/astria-sequencer/src/fees/mod.rs

```diff
@@ -379,17 +370,3 @@ fn base_deposit_fee(asset: &asset::Denom, destination_chain_address: &str) -> u1
     .expect("converting a usize to a u128 should work on any currently existing machine")
     .saturating_add(DEPOSIT_BASE_FEE)
 }
-
-/// Creates `abci::Event` of kind `tx.fees` for sequencer fee reporting
-pub(crate) fn construct_tx_fee_event(fee: &Fee) -> Event {
-    Event::new(
-        "tx.fees",
-        [
-            ("actionName", fee.action_name.to_string()).index(),
-            ("asset", fee.asset.to_string()).index(),
-            ("feeAmount", fee.amount.to_string()).index(),
-            ("sourceTransactionId", fee.source_transaction_id.to_string()).index(),
```

Comment:
The argument for removing the transaction ID is that it will appear in the tx results so that the tx will be identified that way?

Reply:
Our transaction ID is the sha256 of the transaction on-wire bytes. This is the same as